src/pages/community.js

```diff
@@ -0,0 +1,683 @@
+      alt: translateMessageId("page-index-get-started-dapps-image-alt", intl),
+      to: "/community/get-involved/",
+    },
+    {
+      image: data.enterpriseFixed.childImageSharp.fixed,
+      title: "Get support",
+      description:
+        "Need help? Connect with experts from the ecosystem to get your questions answered.",
+      alt: translateMessageId("page-index-get-started-devs-image-alt", intl),
+      to: "/community/support/",
+    },
+  ]
+
+  const whyGetInvolvedCards = [
+    {
+      emoji: ":mage:",
+      title: "Find your tribe",
+      description:
+        "There is a tribe for everyone. Find and connect with like minded individuals to discuss, ponder, and celebrate Ethereum together.",
+    },
+    {
+      emoji: ":dollar:",
+      title: "Earn a living",
+      description:
+        "Everyone has bills to pay. Ethereum allows you to find meaningful work, and get paid well to do it.",
+    },
+    {
+      emoji: ":collision:",
+      title: "Make a difference",
+      description:
+        "Getting involved with Ethereum allows you to be an active stakeholder in a technology that is having a positive impact on millions of people.",
+    },
+  ]
+
+  return (
+    <Page>
+      {/* <PageMetadata
+        title={translateMessageId("page-community-title", intl)}
+        description={translateMessageId(
+          "page-community-meta-description",
+          intl
+        )}
+      /> */}
+      <PageMetadata
+        title="Community home"
+        description="Community homepage description"
+      />
+      <PageHero isReverse content={heroContent} />
+      <Divider />
+      <StyledPurpleContainer>
+        <Content>
+          <CentralColumn>
+            <H2>Why get involved?</H2>
+          </CentralColumn>
+          <CardContainer>
+            {whyGetInvolvedCards.map((card, idx) => (
+              <StyledGetInvolvedCard
+                key={idx}
+                emoji={card.emoji}
+                title={card.title}
+                description={card.description}
+              />
+            ))}
+          </CardContainer>
+        </Content>
+      </StyledPurpleContainer>
+      <StyledGrayContainer>
+        <StyledContent>
+          <IntroRow>
+            <IntroLeftColumn>
+              <H2 id="get-involved">
+                {/* <Translation id="page-community-get-involved" /> */}
+                How can I get involved?
+              </H2>
+              <Subtitle>
+                {/* <Translation id="page-community-get-involved-description" /> */}
+                There are many ways to get involved in the fast-growing Ethereum
+                community; you can join one of the popular online communities,
+                attend an event, join a meetup group, contribute to a project,
+                or participate in one of the many online forums about Ethereum.
+              </Subtitle>
+              <ButtonLink to="/community/grants">
```

Comment:
Another option question - do we want a primary CTA here? 

Perhaps a primary button here to "get involved" & a secondary to "explore online communities"?

Or perhaps we let the user scroll to see the options below?

Reply:
I think just scrolling here is good